.github/workflows/stale.yml

```
@@ -0,0 +1,56 @@
+name: Stale Issues and PRs
+on:
+  schedule:
+  # Process new stale issues once a day. Folks can use /fresh for a fast un-stale
+  # per the commands workflow. Run at 4:15 mostly as a somewhat unique time to
+  # help correlate any issues with this workflow.
+  - cron: '15 4 * * *'
+  workflow_dispatch: {}
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9
+      with:
+        # This action uses ~2 operations per stale issue per run to determine
+        # whether it's still stale. It also uses 2-3 operations to mark an issue
+        # stale or not. During steady state (no issues to mark stale, check, or
+        # close) we seem to use less than 10 operations with ~350 issues and PRs
+        # open.
+        #
+        # Our hourly rate-limit budget for all workflows that use GITHUB_TOKEN
+        # is 1,000 requests per the below docs.
+        # https://docs.github.com/en/rest/overview/resources-in-the-rest-api#requests-from-github-actions
+        operations-per-run: 100
+        # For issues: post a "warning" message after 360 days, then close if another 30 days pass without a response.
+        days-before-issue-stale: 360
+        days-before-issue-close: 30        
```

```suggestion
        # For issues: post a "warning" message after 360 days, then close if another 30 days pass without a response.
        days-before-issue-stale: 90
        days-before-issue-close: 14
```

I feel like mirroring the timings from the crossplane/crossplane repo is a good baseline to start with.

---

Timings have been updated.